tempesta_fw/sock_clnt.c

```diff
@@ -40,6 +40,11 @@
 static struct kmem_cache *tfw_cli_conn_cache;
 static struct kmem_cache *tfw_tls_conn_cache;
 static int tfw_cli_cfg_ka_timeout = -1;
+/*
+ * Maximum size of memory used to store requests. Used for receive window size
+ * steering
+ */
+static size_t recv_window_sz;
```

Comment:
The variable isn't used, it seems you just forgot to do the most important change for https://github.com/tempesta-tech/tempesta/pull/1012#discussion_r187219535